doc/source/ray-core/actors.rst

```diff
@@ -317,6 +317,11 @@ If we instantiate an actor, we can pass the handle around to various tasks.
           std::cout << *counter.Task(&Counter::GetCounter).Remote().Get() << std::endl;
         }
 
+Scheduling the actor
+--------------------
+
+Actors are scheduled similar to tasks, see :ref:`ray-scheduling` for more details.
```

Comment:
Document the zero CPU special case?

Reply:
Also, the fact that by default actors have None as the resource right now. Imo it's different enough from tasks that we can also give it it's own page for clarity, like we have a dedicated fault tolerance page for each concept.